.travis.yml

```diff
@@ -4,7 +4,7 @@
 # Copyright (c) 2013 Mateusz Loskot <mateusz@loskot.net>
 #
 language: cpp
-
+sudo: required
```

Comment:
It was side fix targeted issue discovered here https://travis-ci.org/snikulov/soci/jobs/252953339